docs/examples/compute/instance_chef/block.tf

```diff
@@ -0,0 +1,14 @@
+resource "oci_core_volume" "TFBlock0" {
```

Comment:
I would leave out the block volume part, since that doesn't really have anything to do wth the Chef example, and we show how to attach and mount a volume in other examples.